src/index.js

```diff
@@ -0,0 +1,39 @@
+import './style.css';
+import Tasks from './functionalities.js';
+
+const tasksList = new Tasks();
+
+const newTaskInput = document.querySelector('.enter-task');
+const tasksUL = document.querySelector('.tasks-list');
+const clearBtn = document.querySelector('.clear-btn>button');
+
+window.addEventListener('load', () => {
+  if (localStorage.getItem('tasks')) {
+    tasksList.tasksList = JSON.parse(localStorage.getItem('tasks'));
+    if (tasksList.tasksList.length !== 0) {
+      tasksList.updateDisplay();
+    }
+  }
+});
+
+newTaskInput.addEventListener('keypress', (e) => {
+  if (!e) e = window.event;
```

Comment:
I believe the Lines below are enough to execute the task in this code block. I believe that You Aren't Gonna Need It **(YAGNI)**. This will also ensure you keep it simple and stupid **(KISS)**